src/devices/Mlx90640/README.md

```diff
@@ -0,0 +1,2 @@
+﻿# MLX90640 - Infra Red Camera
+The MLX90614 is an 32x24 pixel Infra Red camera for noncontact temperature measurements
```

Comment:
you should add the usage, reference on the documentation, example of usage (even simple)
Also, in Markdown, there is always an extra line after all titles